### PR TITLE
98 break

### LINKED ISSUE #98: Add `break since start` field to `timetrace status`
The `timetrace status` command prints how long I've been working today. It would be nice to see how long I've been taking breaks since the beginning of my day. For example:

```
timetrace status
+-------------------+----------------------+----------------+--------+
|  CURRENT PROJECT  |  WORKED SINCE START  |  WORKED TODAY  | BREAKS |
+-------------------+----------------------+----------------+--------+
| web-store         | 0h 3min              | 4h 15min       | 30min  |
+-------------------+----------------------+----------------+--------+
```

### PR DESCRIPTION
**Pull request by rknizzle**
Adds break time for the day when you run the status command.
The break time is calculated as the time between all records

Resolves #98 

Screen Shot 2021-05-31 at 11 28 38 PM

### CODE REVIEW

File: core/record.go [hunk 1]
```diff
@@ -158,6 +158,31 @@ func (t *Timetrace) loadAllRecords(date time.Time) ([]*Record, error) {
 	return records, nil
 }
 
+func (t *Timetrace) loadAllRecordsSortedAscending(date time.Time) ([]*Record, error) {
```
Comment (rknizzle) [inline core/record.go:161]: I I decided to create a new function to return the sorted records that breakTime() uses. Let me know if this doesnt seem like a good way to handle getting sorted records. I'm also open to better naming suggestions for loadAllRecordsSortedAscending.

Comment (dominikbraun) [reply core/record.go:161]: Nice! 👍

File: core/timetrace.go [hunk 2]
```diff
@@ -189,6 +211,12 @@ func (report *Report) FormatCurrentTime() string {
 	return formatDuration(*report.TrackedTimeCurrent)
 }
 
+// FormatBreakTime returns the formated string of the total time
+// taking breaks today following the format convention
+func (report *Report) FormatBreakTime() string {
+	return formatDuration(report.BreakTimeToday)
+}
+
```
Comment (dominikbraun) [inline core/timetrace.go:219]: Those formatting-related functions should live in `formatter.go` from now on. There's also #101 to move those existing functions.

**Review by dominikbraun — CHANGES_REQUESTED** [review]
Looks pretty good to me, I just requested a minor change.

File: core/timetrace.go (hunk 2 above)
Comment (rknizzle) [reply core/timetrace.go]: I moved the Format* functions into `formatter.go` in commit 3536a41. They are now methods of Formatter and take a `*Report` as an arg.

**Review by dominikbraun — APPROVED** [review]
LGTM, thanks for implementing this! 👍

**Comment by dominikbraun** [comment]
This PR is contained in [release 0.8.0](https://github.com/dominikbraun/timetrace/releases/tag/v0.8.0).

**Pull request merged**